EIPS/eip-9999.md

```diff
@@ -0,0 +1,191 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 8013
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.